PCbuild/python.props

```diff
@@ -48,6 +48,7 @@
     <ExternalsDir Condition="!HasTrailingSlash($(ExternalsDir))">$(ExternalsDir)\</ExternalsDir>
     <sqlite3Dir>$(ExternalsDir)sqlite-3.21.0.0\</sqlite3Dir>
     <bz2Dir>$(ExternalsDir)bzip2-1.0.6\</bz2Dir>
+    <libffiDir>$(ExternalsDir)libffi\</libffiDir>
```

Comment:
```suggestion
    <libffiDir>$(ExternalsDir)libffi-3.3.0-rc0-r1\</libffiDir>
```

Reply:
I'm happy to help.
Thanks for the feedback